### DISCUSSION TITLE
Docs versões

### DISCUSSION THREAD

**Discussion by Igorios** [Grupo-Plantao/demo-repository#6] [Announcements]
https://docs.google.com/document/d/1EwxJLi0Q8Nh3yLj36LQf4u6RAvkzOi0Ml6rcZ8tnBvE/edit?usp=sharing

**Comment by FelipeHardmann** [comment]
Valeu

**Comment by FelipeHardmann** [comment]
https://www.youtube.com/watch?v=gS0uT4Iidj4
